File: Level 1/약수의 개수와 덧셈/solution.js
function solution(left, right) {
  let answer = 0;

  for (let i = left; i <= right; i++) {
    const measure = [];

    for (let j = 1; j <= i; j++) {
      if (i % j === 0) measure.push(j);
    }
    
    measure.length % 2 === 0 ? (answer += i) : (answer -= i);
  }

  return answer;
}

File: Level 1/약수의 개수와 덧셈/solution.py
def solution(left, right):
    # 약수의 개수가 홀수인 수는 제곱수이므로, 제곱수이면 음수로 바꾸고, 아니면 그대로 더한다.
    return sum(n if n**0.5 % 1 else -n for n in range(left, right + 1))
